### PR TITLE
chore(dbt-fal): add new telemetry to dbt-fal

### PR DESCRIPTION
**Pull request by chamini2**
This removes the fal telemetry from #593 and adds new one for the adapter package.

For now it shares the same project API key.

We need to test before merging.

### CODE REVIEW

**Comment by turbo1912** [comment]
@burkaygur new metrics!

**Comment by mederka** [comment]
Looks great 🤩, @chamini2 !

File: adapter/src/dbt/adapters/fal/wrappers.py [hunk 1]
```diff
@@ -7,10 +7,11 @@
 from dbt.parser.manifest import ManifestLoader
 from dbt.clients.jinja import MacroGenerator
 
-from dbt.adapters.fal_experimental.impl import FalAdapterMixin
-from fal.telemetry import telemetry
 from fal.utils import cache_static
```
Comment (mederka) [inline adapter/src/dbt/adapters/fal/wrappers.py:10]: Are we removing fal as a dependency?

Comment (chamini2) [reply adapter/src/dbt/adapters/fal/wrappers.py:10]: yes!

**Review by isidentical — APPROVED** [review]
LGTM 💯

**Pull request merged**